### ISSUE TITLE
Power & sensitivity : MVPA vs univariate 

### ISSUE THREAD

**Issue by JeanLucAnton** [Centre-IRM-INT/GT-MVPA-nilearn#3]
Q : Can MVPA be used simply as a way of improving power/sensitivity over univariate methods i.e. given an appropriate experimental design, can MVPA be used to answer the same kinds of questions we ask using univariate methods?  If the answer is yes, then I guess this means that we all potentially have existing data-sets that could be reanalysed using multivariate methods?  If the experiment was originally designed for a univariate analysis, how can we best/fairly split the data into a training set and a test set ?

**Comment by JeanLucAnton** [comment]
A : The response seems to be yes, as the use of MVPA tends to increase sensitivity over univariate analyses (Li et al., 2009, Neuron). In the field of motor cognition, for example, Buchwald et al. (2018) reanalyzed with MVPA a data-set already analyzed and published in 2017 with univariate analyses. However, I ignore how easy it is to make MVPA on data recorded with a fMRI design originally created for univariate analyses.

**Comment by SylvainTakerkart** [comment]
Hi @JeanLucAnton . I don't know these papers, can you include direct links? (maybe with just the DOI link...)

From a methodological point of view, this question is ill-posed: there are two many factors/parameters that make univariate and decoding analysis different that it's impossible to theoretically warrant that multivariate decoding analysis are more powerful than univariate GLM analyses... They just do different things! But indeed, one hope is that multivariate could be more powerful by "gathering" small effects from different voxels that could become significant by being considered alltogether (while none of them would be significant with a GLM)...
